alacritty/src/window.rs

```diff
@@ -33,14 +33,21 @@ use image::ImageFormat;
 #[cfg(not(any(target_os = "macos", windows)))]
 use x11_dl::xlib::{Display as XDisplay, PropModeReplace, XErrorEvent, Xlib};
 
-use alacritty_terminal::config::{Decorations, StartupMode, WindowConfig};
+use alacritty_terminal::config::{
+    Colors as TerminalColors, Decorations, StartupMode, WindowConfig,
```

Comment:
Why did you rename this import, especially to `TerminalColors`? There is no conflict, right? And if anything, aren't these the config colors?